src/app/GitUI/UserControls/EditboxBasedConsoleOutputControl.cs

```diff
@@ -31,14 +33,14 @@ public EditboxBasedConsoleOutputControl()
         {
             _editbox = new RichTextBox
             {
-                BackColor = SystemColors.Info,
+                BackColor = AppColor.EditorBackground.GetThemeColor(),
```

Comment:
This RichTextBox is read-only. The color was chosen on purpose, refer to 8c702c169aa3f97c39eeec75d5d7810872b8620c.

Reply:
So are most Fileviewers, that has SystemColors.Window by default
But I can change this in dark mode only